db/schema.rb

```diff
@@ -34,6 +34,13 @@
 
   add_index "downvotes", ["user_id", "review_id"], name: "index_downvotes_on_user_id_and_review_id", unique: true, using: :btree
 
+  create_table "game_tags", force: :cascade do |t|
+    t.integer "tag_id",  null: false
+    t.integer "game_id", null: false
+  end
+
+  add_index "game_tags", ["tag_id", "game_id"], name: "index_game_tags_on_tag_id_and_game_id", unique: true, using: :btree
```

Comment:
You probably want an index on `game_id` as well. This index only contains `tag_id` first so it will be used when trying to find all of the games for a particular tag. But if you want to find all of the tags for a particular game, you'll need another index that just includes `game_id`:

```ruby
add_index :game_tags, :game_id
```